ldclient/client.py

```diff
@@ -27,79 +25,10 @@
 from cachecontrol import CacheControl
 from threading import Lock
 
-GET_LATEST_FEATURES_PATH = '/sdk/latest-flags'
```

Comment:
These changes are api-breaking, so we'll be on a new major release anyway. It's one less change our customers will make, but is arguably cruft. I guess the argument is that it's one less thing they will have to change when updating the client.